tests/test_transit_coverage.py

```diff
@@ -1,17 +1,40 @@
 import numpy as np
+from exovetter import transit_coverage
 import pytest
 from numpy.testing import assert_allclose
 
-from exovetter import transit_coverage
+
+def test_coverage1():
+    time = np.arange(0, 100, step=.01)
+    p_day = 3
+    epoch = 3
+    dur_hour = 5
+
+    coverage, h, b = transit_coverage.calc_coverage(
+        time, p_day, epoch, dur_hour, ndur=2, nbins=10)
+
+    assert coverage == 1.0
+
+
+def test_coverage2():
+    time = np.arange(0, 100, step=1)
+    p_day = 3
+    epoch = 3
+    dur_hour = 12
+
+    coverage, h, b = transit_coverage.calc_coverage(
+        time, p_day, epoch, dur_hour, ndur=2, nbins=10)
+
+    assert coverage == 0.1
 
 
 @pytest.mark.parametrize(
     ('step', 'p_day', 'epoch', 'dur_hour', 'ans'),
     [(0.01, 3, 3, 5, 1),
      (1, 3, 3, 12, 0.1),
      (1, 5, 10.25, 24, 0.2)])
-def test_coverage1(step, p_day, epoch, dur_hour, ans):
+def test_coverage3(step, p_day, epoch, dur_hour, ans):
```

Comment:
I think your new ` test_coverage1` and ` test_coverage2` can be folded into the original function pytest parameterization to avoid much code duplication.

Reply:
Oh wow, I hadn't realized that you had grouped them all together into this one test.  
Fancy, but quite hard to read.  
does it loop over all three arrays?

Reply:
Yes, it runs the test 3x.

1. `step, p_day, epoch, dur_hour, ans` -> `0.01, 3, 3, 5, 1`
2. `step, p_day, epoch, dur_hour, ans` -> `1, 3, 3, 12, 0.1`
3. `step, p_day, epoch, dur_hour, ans` -> `1, 5, 10.25, 24, 0.2`

Also see https://docs.pytest.org/en/stable/parametrize.html . Sorry about the hard to read part, but in the long run, it saves some work. 😬

Reply:
I can see how for a lots of tests this is very useful...